src/server/pfs/server/server_test.go

```diff
@@ -81,6 +81,16 @@ func TestInvalidRepo(t *testing.T) {
 	t.Parallel()
 	client, _ := getClientAndServer(t)
 	require.YesError(t, client.CreateRepo("/repo"))
+
+	require.NoError(t, client.CreateRepo("lenny"))
+	require.NoError(t, client.CreateRepo("lenny123"))
+	require.NoError(t, client.CreateRepo("lenny_123"))
+
+	require.YesError(t, client.CreateRepo("lenny-123"))
+	require.YesError(t, client.CreateRepo("lenny.123"))
+	require.YesError(t, client.CreateRepo("lenny:"))
+	require.YesError(t, client.CreateRepo("lenny,"))
+	require.YesError(t, client.CreateRepo("lenny#"))
```

Comment:
Assuming this is supposed to be a reference to our canine friend it's actually leni... Needless to say this is not a real defect in this code review.